server/auth/sso/sso.go

```diff
@@ -181,25 +183,41 @@ func newSso(
 	if err != nil {
 		return nil, fmt.Errorf("failed to create JWT encrpytor: %w", err)
 	}
+
+	var filterGroupsRegex []*regexp.Regexp
+	if c.FilterGroupsRegex != nil && len(c.FilterGroupsRegex) > 0 {
+		for _, regex := range c.FilterGroupsRegex {
+			compiledRegex, err := regexp.Compile(regex)
+			if err != nil {
+				return nil, fmt.Errorf("failed to compile sso.filterGroupRegex: %s %w", regex, err)
+			}
+			filterGroupsRegex = append(filterGroupsRegex, compiledRegex)
+		}
+	}
+
 	lf := log.Fields{"redirectUrl": config.RedirectURL, "issuer": c.Issuer, "issuerAlias": "DISABLED", "clientId": c.ClientID, "scopes": config.Scopes, "insecureSkipVerify": c.InsecureSkipVerify}
 	if c.IssuerAlias != "" {
 		lf["issuerAlias"] = c.IssuerAlias
 	}
+	if c.FilterGroupsRegex != nil && len(c.FilterGroupsRegex) > 0 {
+		lf["filterGroupsRegex"] = c.FilterGroupsRegex
+	}
```

Comment:
I believe this can just be added to `lf` directly. `issuerAlias` is the odd one out since it logs out "DISABLED", whereas the rest log out the field directly.

`scopes` above is similarly a `[]string` (i.e. same as `filterGroupsRegex`), with regard to typing

Reply:
done